src/Parser/Warrior/Fury/Modules/BuffDebuff/EnrageUptime.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+import Analyzer from 'Parser/Core/Analyzer';
+
+import SPELLS from 'common/SPELLS';
+import SpellIcon from 'common/SpellIcon';
+import SpellLink from 'common/SpellLink';
+import { formatPercentage } from 'common/format';
+import StatisticBox from 'Main/StatisticBox';
+import Combatants from 'Parser/Core/Modules/Combatants';
+
+class enrageUptime extends Analyzer {
+  static dependencies = {
+    combatants: Combatants,
+  };
+
+  suggestions(when) {
+    const enrageUptime = this.combatants.selected.getBuffUptime(SPELLS.ENRAGE.id) / this.owner.fightDuration;
+
+    when(enrageUptime).isLessThan(0.70)
+      .addSuggestion((suggest, actual, recommended) => {
+        return suggest(<span>Your <SpellLink id={SPELLS.ENRAGE.id} /> uptime can be improved.</span>)
```

Comment:
Please use the `Wrapper` component from `common/Wrapper` instead of `span` when you don't actually need the container.